docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,119 @@
 
 ## Known Issues for PowerShell on Non-Windows Platforms
 
-The first Alpha release of PowerShell on Linux and macOS is mostly functional but
-does have some significant limitations and usability issues.
+Alpha releases of PowerShell on Linux and macOS are mostly functional but do have some significant limitations and usability issues.
```

Comment:
I'm not sure the best way to achieve the linking (is it to the latest alpha milestone? The general beta milestone?), but I did add a line here about the higher quality of Beta releases (as I expect we'll have alphas again in the future for future versions post 6.0).